pages/profile/read/views/profile.jsx

```diff
@@ -199,7 +200,11 @@ class Profile extends React.Component {
                 )
               }
               {
-                (isOwnProfile || allowedActions.includes('pil.create')) && !pilActive && over18 && (
+                (isOwnProfile || allowedActions.includes('pil.create')) &&
+                  !pilActive &&
+                  over18 &&
+                  (correctEstablishment || !pilIncomplete) &&
```

Comment:
This is getting pretty burly now. Perhaps pull it out into a `const canApply`?

Reply:
I thought you might have :cow2: with that one